llumnix/arg_utils.py

```diff
@@ -70,13 +70,15 @@ def create_engine_manager_configs(
         return global_scheduler_config
 
     def create_migration_configs(
-        self,
+        self, instance_rank_map, pp_or_tp_enabled
```

Comment:
instance_rank_map -> instance_rank, the name of most variables in Llumnix and vLLM don't directly show the data structure.